go-controller/pkg/ovn/egressip.go

```diff
@@ -567,14 +567,24 @@ func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool)
 
 func (oc *Controller) addEgressNode(egressNode *kapi.Node) error {
 	klog.V(5).Infof("Egress node: %s about to be initialized", egressNode.Name)
-	if stdout, stderr, err := util.RunOVNNbctl(
-		"set",
-		"logical_switch_port",
-		types.EXTSwitchToGWRouterPrefix+types.GWRouterPrefix+egressNode.Name,
-		"options:nat-addresses=router",
-	); err != nil {
+	lsp := nbdb.LogicalSwitchPort{
+		Name:    types.EXTSwitchToGWRouterPrefix + types.GWRouterPrefix + egressNode.Name,
+		Options: map[string]string{"nat-addresses": "router"},
+	}
+	opModel := libovsdbops.OperationModel{
+		Model: &lsp,
+		ModelPredicate: func(lsp *nbdb.LogicalSwitchPort) bool {
+			return lsp.Name == types.EXTSwitchToGWRouterPrefix+types.GWRouterPrefix+egressNode.Name
+		},
```

Comment:
The logical switch port is indexed by name, so you shouldn't need a `ModelPredicate` to lookup the model.

Reply:
done